src/Main.java

```diff
@@ -46,9 +46,9 @@ public static void main(String[] args) {
         System.out.println("Всего работников в компании " + totalEmployees + " человек");
         var employees = 80;
         var newEmployees = 94;
```

Comment:
не нужно было заводить новую переменную, нужно было просто поменять значение существующей:
```suggestion
        totalEmployees += 94;
```